public/projects/guess.md

```diff
@@ -0,0 +1,39 @@
+# Guess The Number
+
+In this game, the *human* thinks of a number between 1 and 100 and the *computer* tries to guess it.
+
+Educationally speaking, this is a good example of a *binary search algorithm* (not to be confused with *binary notation* which is a different kind of binary). If the programmer does their job, then the game will find the correct number in no more than [log2(n)+1] guesses -- so for n=100, that's 7 guesses *max*.
```

Comment:
It may be useful to begin this exercise without using binary search, and allow the students to come to an understanding of why binary search is an improvement after implementing exhaustive search first.

This would allow the students to feel the difference in the performance of the two approaches acutely.